dataService.js

```diff
@@ -0,0 +1,53 @@
+// Getting data from json files
+const categoryData = require('./src/data/categories.json');
```

Comment:
Don't load this data in advance. Load it when you need inside the required methods.

Reply:
Done!